src/library/simplified/extension/cesc-LT-channel-country-network-DTL.js

```diff
@@ -5,6 +5,9 @@ var priorTimeStamp = '0';
 var priorMarketingCode = "";
 var length = marketingChannels.length;
 
+
+// Old logic that goes through cesc fields to find last touch and then set DTL.
```

Comment:
This comment has always been there. This PR showed that I added it but I didn't. I guess things get shifted around when I copy the code to a text editor to modify and then copy it back to the extension.